com.github.mimillieuh.thinkercad-desktop.yml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.mimillieuh.thinkercad-desktop
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: com.github.mimillieuh.thinkercad-desktop
+rename-desktop-file: ThinkerCAD.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download
+modules:
+  - name: ThinkerCAD-Desktop
+    buildsystem: simple
+    build-commands:
+      - unzip ThinkerCAD-linux-x64.zip
+
+      - install -dm755 "${FLATPAK_DEST}/bin" "${FLATPAK_DEST}/lib/ThinkerCAD-linux-x64"
+      - cp -r ThinkerCAD-linux-x64/* "${FLATPAK_DEST}/lib/ThinkerCAD-linux-x64"
+
+      - install -Dm644 "com.github.mimillieuh.thinkercad-desktop.png" "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.github.mimillieuh.thinkercad-desktop.png"
+      - install -Dm755 "run.sh" "${FLATPAK_DEST}/bin/com.github.mimillieuh.thinkercad-desktop"
+      - desktop-file-install --dir="${FLATPAK_DEST}/share/applications" --remove-key=X-AppImage-Version --set-key=Exec --set-value=com.github.mimillieuh.thinkercad-desktop --set-key=Icon --set-value=com.github.mimillieuh.thinkercad-desktop ThinkerCAD.desktop
+
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo com.github.mimillieuh.thinkercad-desktop.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/MiMillieuh/ThinkerCAD-Desktop/releases/download/1.0.0/ThinkerCAD-linux-x64.zip
```

Comment:
Does this electron runtime address all the 0-day vulnerabilities disclosed in the last couple of weeks like in webP ?

Reply:
Electron 26.2.2